macros/cargo

```diff
@@ -16,11 +16,14 @@
 %__cargo_cross_opts %{__cargo_common_opts} --target %{__cargo_target}
 %__cargo_cross_opts_static %{__cargo_common_opts} --target %{__cargo_target_static}
 %__cargo_env CARGO_TARGET_DIR="${HOME}/.cache" SKIP_README="true"
+%__cargo_env_fips CARGO_TARGET_DIR="${HOME}/.cache/.fips" SKIP_README="true"
```

Comment:
It seems like you also need `__cargo_env_fips_static` (and related other macros) for cases like `apiclient` where both FIPS and static are in use.

Reply:
Yeah, I wanted to keep that out from the initial review, I have it locally.